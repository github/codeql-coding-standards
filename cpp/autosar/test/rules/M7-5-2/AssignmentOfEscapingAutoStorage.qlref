rules/M7-5-2/AssignmentOfEscapingAutoStorage.ql