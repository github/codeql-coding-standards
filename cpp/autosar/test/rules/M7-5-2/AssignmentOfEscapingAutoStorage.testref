cpp/common/test/rules/donotcopyaddressofautostorageobjecttootherobject/DoNotCopyAddressOfAutoStorageObjectToOtherObject.ql