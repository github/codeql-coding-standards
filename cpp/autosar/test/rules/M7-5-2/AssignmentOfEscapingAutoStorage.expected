| manager.cpp:48:3:48:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | manager.cpp:48:15:48:17 | man | source |
| stack_escapes_test.cpp:19:3:19:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:19:13:19:13 | x | source |
| stack_escapes_test.cpp:27:3:27:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:27:13:27:13 | x | source |
| stack_escapes_test.cpp:34:3:34:12 | ... = ... | A stack address which arrived via a $@ may be assigned to a non-local variable. | stack_escapes_test.cpp:34:12:34:12 | p | parameter |
| stack_escapes_test.cpp:68:3:68:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:62:14:62:14 | x | source |
| stack_escapes_test.cpp:78:3:78:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:78:14:78:14 | s | source |
| stack_escapes_test.cpp:88:3:88:14 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:84:13:84:13 | x | source |
| stack_escapes_test.cpp:113:3:113:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:113:13:113:13 | x | source |
| stack_escapes_test.cpp:122:3:122:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:122:13:122:13 | x | source |
| stack_escapes_test.cpp:131:3:131:11 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:131:11:131:11 | x | source |
| stack_escapes_test.cpp:156:3:156:11 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:156:10:156:11 | b1 | source |
| stack_escapes_test.cpp:158:3:158:15 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:158:11:158:12 | b1 | source |
| stack_escapes_test.cpp:161:3:161:11 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:161:10:161:11 | b2 | source |
| stack_escapes_test.cpp:163:3:163:15 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:163:11:163:12 | b2 | source |
| stack_escapes_test.cpp:165:3:165:14 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:165:10:165:11 | b2 | source |
| stack_escapes_test.cpp:167:3:167:18 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:167:11:167:12 | b2 | source |
| stack_escapes_test.cpp:170:3:170:11 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:170:10:170:11 | b3 | source |
| stack_escapes_test.cpp:172:3:172:15 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:172:11:172:12 | b3 | source |
| stack_escapes_test.cpp:174:3:174:14 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:174:10:174:11 | b3 | source |
| stack_escapes_test.cpp:176:3:176:18 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:176:11:176:12 | b3 | source |
| stack_escapes_test.cpp:178:3:178:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:178:10:178:11 | b3 | source |
| stack_escapes_test.cpp:180:3:180:21 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:180:11:180:12 | b3 | source |
| stack_escapes_test.cpp:183:3:183:14 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:183:13:183:14 | b1 | source |
| stack_escapes_test.cpp:185:3:185:18 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:185:14:185:15 | b1 | source |
| stack_escapes_test.cpp:187:3:187:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:187:13:187:14 | b2 | source |
| stack_escapes_test.cpp:189:3:189:21 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:189:14:189:15 | b2 | source |
| stack_escapes_test.cpp:191:3:191:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:191:13:191:14 | b3 | source |
| stack_escapes_test.cpp:193:3:193:24 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:193:14:193:15 | b3 | source |
| stack_escapes_test.cpp:224:3:224:14 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:224:13:224:14 | b1 | source |
| stack_escapes_test.cpp:226:3:226:18 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:226:14:226:15 | b1 | source |
| stack_escapes_test.cpp:228:3:228:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:228:13:228:14 | b2 | source |
| stack_escapes_test.cpp:230:3:230:21 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:230:14:230:15 | b2 | source |
| stack_escapes_test.cpp:232:3:232:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:232:13:232:14 | b3 | source |
| stack_escapes_test.cpp:234:3:234:24 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:234:14:234:15 | b3 | source |
| stack_escapes_test.cpp:242:3:242:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:242:10:242:10 | x | source |
| stack_escapes_test.cpp:244:3:244:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:244:11:244:11 | x | source |
| stack_escapes_test.cpp:247:3:247:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:247:10:247:10 | x | source |
| stack_escapes_test.cpp:249:3:249:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:249:11:249:11 | x | source |
| stack_escapes_test.cpp:251:3:251:16 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:251:10:251:10 | x | source |
| stack_escapes_test.cpp:253:3:253:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:253:11:253:11 | x | source |
| stack_escapes_test.cpp:256:3:256:13 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:256:10:256:10 | x | source |
| stack_escapes_test.cpp:258:3:258:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:258:11:258:11 | x | source |
| stack_escapes_test.cpp:260:3:260:16 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:260:10:260:10 | x | source |
| stack_escapes_test.cpp:262:3:262:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:262:11:262:11 | x | source |
| stack_escapes_test.cpp:264:3:264:19 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:264:10:264:10 | x | source |
| stack_escapes_test.cpp:266:3:266:23 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:266:11:266:11 | x | source |
| stack_escapes_test.cpp:271:3:271:16 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:271:13:271:13 | x | source |
| stack_escapes_test.cpp:273:3:273:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:273:14:273:14 | x | source |
| stack_escapes_test.cpp:275:3:275:19 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:275:13:275:13 | x | source |
| stack_escapes_test.cpp:277:3:277:23 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:277:14:277:14 | x | source |
| stack_escapes_test.cpp:279:3:279:22 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:279:13:279:13 | x | source |
| stack_escapes_test.cpp:281:3:281:26 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:281:14:281:14 | x | source |
| stack_escapes_test.cpp:319:3:319:16 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:319:13:319:13 | x | source |
| stack_escapes_test.cpp:321:3:321:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:321:14:321:14 | x | source |
| stack_escapes_test.cpp:323:3:323:19 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:323:13:323:13 | x | source |
| stack_escapes_test.cpp:325:3:325:23 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:325:14:325:14 | x | source |
| stack_escapes_test.cpp:327:3:327:22 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:327:13:327:13 | x | source |
| stack_escapes_test.cpp:329:3:329:26 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:329:14:329:14 | x | source |
| stack_escapes_test.cpp:364:3:364:14 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:364:14:364:14 | x | source |
| stack_escapes_test.cpp:365:3:365:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:365:17:365:17 | x | source |
| stack_escapes_test.cpp:366:3:366:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:366:20:366:20 | x | source |
| stack_escapes_test.cpp:367:3:367:17 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:367:17:367:17 | x | source |
| stack_escapes_test.cpp:368:3:368:20 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | stack_escapes_test.cpp:368:20:368:20 | x | source |
| test.cpp:7:5:7:10 | ... = ... | A stack address ($@) may be assigned to a non-local variable. | test.cpp:7:10:7:10 | c | source |
